scripts/push-docker.sh

```diff
@@ -3,18 +3,9 @@
 # Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 # See LICENSE.txt for license information.
 
-set -e
-set -u
+set -euox
 
-if [[ -z "${CIRCLE_TAG:-}" ]]; then
-  echo "Pushing lastest for $CIRCLE_BRANCH..."
-  TAG=latest
-else
-  echo "Pushing release $CIRCLE_TAG..."
-  TAG="$CIRCLE_TAG"
-fi
-
-echo $DOCKER_PASSWORD | docker login --username $DOCKER_USERNAME --password-stdin
+echo $DOCKER_TOKEN | docker login --username $DOCKER_USERNAME --password-stdin
 
 docker tag mattermost/mattermost-cloud:test mattermost/mattermost-cloud:$TAG
 docker tag mattermost/mattermost-cloud-e2e:test mattermost/mattermost-cloud-e2e:$TAG
```

Comment:
Would vote in favor of killing all bash scripts laying around the project. and encapsulate this functionality on a root Makefile.

Reply:
Resolving since it is out of the scope. We are only migrating to ghactions in this iteration. 
Makefile concerns to be addressed in next iteration.